examples/js/archive.js

```diff
@@ -89,8 +108,10 @@ const renderImages = (fullResImages, url) => {
 // renders thumbnails or images in thumbnail size
 const renderThumbnails = (thumbnails = [], url, fullResImgs) => {
   const imagesToRender = thumbnails || fullResImgs;
+  const paginatedImages = currPagination.paginate(imagesToRender);
+  imageCount = imagesToRender.length;
 
-  imagesToRender.forEach((file) => {
+  paginatedImages[currPage].forEach((file) => {
```

Comment:
Ahhhh i see. So this is an array which contains a sub-array for each page. Could we instead have it be a method of Paginate? So, you'd say, `currPagination.imagesForPage(currPage)`, maybe? 

Then we'd manage the whole array inside of the Pagination class, and the outside world won't have to know the details of how it's stored, just how to ask for the current set?

Reply:
Hmm, I'll check it out